cdc/model/codec/v1/codec.go

```diff
@@ -101,7 +103,7 @@ type DDLEvent struct {
 	TableInfo    *model.TableInfo   `msg:"-"`
 	PreTableInfo *model.TableInfo   `msg:"-"`
 	Type         timodel.ActionType `msg:"-"`
-	Done         bool               `msg:"-"`
+	Done         atomic.Bool        `msg:"-"`
```

Comment:
There is no need to use atomic here because the field is protected by a lock in ddl sink.

Reply:
![Uploading image.png…]()

Reply:
we need to read it else where.